trestle/transforms/implementations/xccdf.py

```diff
@@ -26,14 +26,18 @@
 from ruamel.yaml import YAML
 
 from trestle.oscal.assessment_results import AssessmentAssets
-from trestle.oscal.assessment_results import ControlSelection
 from trestle.oscal.assessment_results import LocalDefinitions1
 from trestle.oscal.assessment_results import Observation
 from trestle.oscal.assessment_results import Result
-from trestle.oscal.assessment_results import ReviewedControls
-from trestle.oscal.assessment_results import Status1
 from trestle.oscal.assessment_results import SystemComponent
-from trestle.oscal.common import AssessmentPlatform, ImplementedComponent, InventoryItem, Property, SubjectReference
+from trestle.oscal.common import AssessmentPlatform
```

Comment:
Agree with Jenn

Reply:
```
(venv.fix-1.1.2) degenaro:trestle.fix-1.1.2$ make code-lint
pre-commit run flake8 --all-files
flake8...................................................................Passed
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

trestle/transforms/implementations/xccdf.py:29:121: E501 line too long (160 > 120 characters)
```
Because `from trestle.oscal.common import` all on one line failed the linter, I chose to make all the imports singletons for consistency. I can make those that are less than 120 into one line if you like. I'm not too fussy here.